db/seeds.rb

```diff
@@ -6,5 +6,7 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 User.create(email: 'admin@sverigeshundforetagare.se', password: 'hundapor', admin: true)
-BusinessCategory.create(name:'Groomer')
-BusinessCategory.create(name: 'Trainer')
+
+
+business_categories = ['Träning', 'Psykologi', 'Rehab', 'Butik', 'Trim', 'Friskvård', 'Dagis', 'Pensionat', 'Rastare', 'Sociala tjänstehundar (vårdhund mm)', 'Civila tjänstehundar (med en förare)', 'Skola']
```

Comment:
So:

remove 'Rastare',

Change
'Sociala tjänstehundar (vårdhund mm)', 'Civila tjänstehundar (med en förare)', 

to 
name: 'Sociala tjänstehundar'
desc:  'Terapi-, vård- & skolhund dvs hundar som jobbar tillsammans med sin förare/ägare inom vård, skola och omsorg.'
and
name: 'Civila tjänstehundar'
desc: 'Assistanshundar dvs hundar som jobbar åt sin ägare som service-, signal, diabetes, PH-hund mm'